tests/testColumnOrigins.py

```diff
@@ -6,11 +6,11 @@
 from lsst.sims.catalogs.measures.instance import InstanceCatalog, cached, compound
 from lsst.sims.catalogs.generation.db import CatalogDBObject
 
-def makeTestDB(size=10, **kwargs):
+def makeTestDB(size=10, name=None, **kwargs):
     """
     Make a test database
```

Comment:
Since this is not a 'private' function, should arguments be explained in the doc strings?